src/api/routers/participantsRouter.ts

```diff
@@ -266,25 +269,107 @@ export function createParticipantsRouter() {
     }
   );
 
-  const sharingRelationParser = z.object({
-    newParticipantSites: z.array(z.number()),
-  });
-
   participantsRouter.get(
-    '/:participantId/apiKeys',
+    '/:participantId/apiKeys/list',
     async (req: ParticipantRequest, res: Response) => {
       const { participant } = req;
       if (!participant?.siteId) {
         return res.status(400).send('Site id is not set');
       }
 
-      const adminApiKeys = await getApiKeys(participant.siteId);
+      const adminApiKeys = await getApiKeysFromAdmin(participant.siteId);
       const apiKeys = await mapAdminApiKeysToApiKeyDTOs(adminApiKeys);
 
       return res.status(200).json(apiKeys);
     }
   );
 
+  const apiKeyIdParser = z.object({
+    keyId: z.string(),
+  });
+  participantsRouter.get(
+    '/:participantId/apiKeys',
```

Comment:
would it make sense to change this to the singular `/:participantId/apiKey`?

Reply:
Yeah this makes sense to me. Will adjust the routes to match. I would think then the listing one could just be `/:participantId/apiKeys`